bootstrap.go

```diff
@@ -147,38 +154,107 @@ func checkBootstrapped(s Session) (bool, error) {
 	return v, nil
 }
 
-func checkBootstrappedVar(s Session) (bool, error) {
+// Get variable value from mysql.tidb table.
+// Those variables are used by TiDB server.
+func getTiDBVar(s Session, name string) (types.Datum, error) {
 	sql := fmt.Sprintf(`SELECT VARIABLE_VALUE FROM %s.%s WHERE VARIABLE_NAME="%s"`,
-		mysql.SystemDB, mysql.TiDBTable, bootstrappedVar)
+		mysql.SystemDB, mysql.TiDBTable, name)
 	rs, err := s.Execute(sql)
 	if err != nil {
-		if infoschema.ErrTableNotExists.Equal(err) {
-			return false, nil
-		}
-		return false, errors.Trace(err)
+		return types.Datum{}, errors.Trace(err)
 	}
-
 	if len(rs) != 1 {
-		return false, errors.New("Wrong number of Recordset")
+		return types.Datum{}, errors.New("Wrong number of Recordset")
 	}
 	r := rs[0]
 	row, err := r.Next()
 	if err != nil || row == nil {
-		return false, errors.Trace(err)
+		return types.Datum{}, errors.Trace(err)
 	}
+	return row.Data[0], nil
+}
 
-	isBootstrapped := row.Data[0].GetString() == bootstrappedVarTrue
+func checkBootstrappedVar(s Session) (bool, error) {
```

Comment:
Since `getTiDBVar` is extracted, this function has similar name as `checkBootstrapped` which is confusing.
How about put the code in this function into `checkBootstrapped`?